tests/test_manager_graph.py

```diff
@@ -6,6 +6,7 @@
 import os
 import time
 import unittest
+import sqlalchemy.exc
```

Comment:
Have you optimized your imports with PyCharm to make sure they're all formatted properly?